raiden/ui/app.py

```diff
@@ -336,14 +337,23 @@ def run_app(
         f'node_{pex(address)}',
         f'netid_{given_numeric_network_id}',
         f'network_{pex(token_network_registry.address)}',
-        f'v{RAIDEN_DB_VERSION}_log.db',
     )
-    config['database_path'] = database_path
+
+    config['database_path'] = os.path.join(database_path, f'v{RAIDEN_DB_VERSION}_log.db')
+
+    log_file_path = os.path.join(database_path, 'debug.log')
+    log_file_path = kwargs.get('log_file') if kwargs.get('log_file') else log_file_path
+    configure_logging(
+        kwargs.get('log_config'),
+        log_json=kwargs.get('log_json'),
+        log_file=log_file_path,
```

Comment:
The correct parameter is `debug_log_file_name`.

Reply:
should the ```log_file``` param still be assigned something or just ```debug_log_file_name=log_file_path ```?